packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -110,11 +120,16 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
   > = "multiple";
 
   /**
-   * Specifies the size of the component inherited from the `calcite-combobox`, defaults to `m`.
+   * The component's short heading.
    *
-   * @internal
+   * When provided, the short heading will be displayed in the component's selection.
+   *
+   * It is recommended to use 5 characters or fewer.
    */
-  @Prop() scale: Scale = "m";
+  @Prop({ reflect: true }) shortHeading: string;
```

Comment:
Should we have `reflects` tests for description and shortHeading? Probably don't need `defaults` since these are both undefined by default.

Reply:
I'll submit a separate PR for this since the test doesn't have `reflects` set up yet.